obal/data/roles/diff_package/tasks/copr.yml

```diff
@@ -0,0 +1,33 @@
+---
+- include_tasks: git_package_info.yml
+
+- name: 'Get latest package info from copr'
+  command: >
+    copr-cli
+      get-package
+      {{ copr_user }}/{{ diff_package_copr_repo }}
+      --name {{ inventory_hostname }}
+      --with-latest-build
+  register: copr_package_info
+  ignore_errors: true
+
+- set_fact:
+    copr_package_info: "{{ copr_package_info.stdout | from_json }}"
+  when: copr_package_info | success
+
+- set_fact:
+    copr_package_version: "{{ copr_package_info['name'] + '-' + copr_package_info['latest_build']['pkg_version'] }}"
+  when: copr_package_info | success
+
+- set_fact:
+    copr_package_version: ""
+  when: copr_package_info | failed
+
+- set_fact:
+    diff_package_changed: "{{ git_package_version not in copr_package_version }}"
```

Comment:
Should this be a check for "startswith" (`{{ not copr_package_version.startswith(git_package-version) }}`)?